code/modules/admin/admin.dm

```diff
@@ -273,6 +273,15 @@ var/global/nologevent = 0
 
 	show_note(key)
 
+/datum/admins/proc/vpn_whitelist()

```

Comment:
Does this show the current whitelisted keys and reasons?

Reply:
No.
There's not much point in doing so, because:

- Existing entries are quite likely to become irrelevant within a week, since anyone who plays for 10+ hours no longer needs a whitelist entry.
- I'd rather make the list available via a web-based tool, if we have to have a list.
- There's never really any cause to remove someone from the list. If someone on the list breaks the rules, we just ban them.